docs/UserGuide.md

```diff
@@ -129,6 +146,15 @@ Examples:
 *  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
 *  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
 
+> [! NOTE]
```

Comment:
Please fix this, should be `[!Note]`

Reply:
Keen eye! Thank you for spotting the error!